django_roa/db/query.py

```diff
@@ -161,6 +162,68 @@ def has_results(self, *args, **kwargs):
         return True
 
 
+class ROAModelIterable(BaseIterable):
+    """
+    Iterator that yields a model instance for each row of a ROAModel.
+    """
+
+    def __iter__(self):
+        queryset = self.queryset
+        query = queryset.query
+        try:
+            parameters = query.parameters
+            logger.debug("""Retrieving : "%s" through %s with parameters "%s" """ % (
+                queryset.model.__name__,
+                queryset.model.get_resource_url_list(),
+                force_text(parameters)))
+            if ROA_SSL_CA:
+                response = queryset._get_requests_client().get(
+                    queryset.model.get_resource_url_list(),
+                    params=parameters,
+                    headers=queryset._get_http_headers(),
+                    verify=ROA_SSL_CA)
+            else:
+                response = queryset._get_requests_client().get(
+                    queryset.model.get_resource_url_list(),
```

Comment:
nit: Just wondering if we can avoid calling this function multiple times as it's already called in debug log statement.